### ISSUE TITLE
スマートフォン向けの標準デザイン

### ISSUE THREAD

**Issue by tdtds** [tdiary/tdiary-core#71]

**Comment by tdtds** [comment]
昨今はmedia queryを使うことでthemeだけでレスポンシブにできるようになってきたのと(defaultテーマ参照: beb59167482b0f7c9bf58981ca581e2f02599c20)、サーバサイドで切り分ける手段をなくしたので(#507)、この要件はほぼなくなったと考えています。

ただ各テーマで対応する必要があるけど、一部をbase.cssに持ってくることは可能かも知れない。

**Comment by machu** [comment]
スマートフォンでもいい感じに読み書きできる標準的なテーマがあるといいなとは思っています。

**Comment by tdtds** [comment]
最新のdefaultテーマはわりといい感じだと思いますが、どうですか?

**Comment by machu** [comment]
naviのリンクをもう少し大きくしたいのと、今は非そうですね。あとは、表示にしているsidebarからの検索やカレンダーからの過去日記へのアクセスができればと思ってます。

**Comment by machu** [comment]
そうですね。あとは、naviのリンクをもう少し大きくしたいのと、今は表示にしているsidebarからの検索やカレンダーからの過去日記へのアクセスができればと思ってます。

**Comment by tdtds** [comment]
naviはそのとおり。ちょっと調整してみます。

サイドバーのアイテムはハンバーガーメニューに押し込めるべきだとは思うけど、テーマだけでやるのはヘビーかなぁ。
